README.md

```diff
@@ -51,12 +51,19 @@ For greenfield projects you might want to consider to use [mobx-react-lite](http
 
 Please check [mobx.js.org](https://mobx.js.org) for the general documentation. The documentation below highlights some specifics.
 
-### `observer(componentClass)`
+### `observer(component)`
 
-Function (and decorator) that converts a React component definition, React component class or stand-alone render function into a reactive component, which tracks which observables are used by `render` and automatically re-renders the component when one of these values changes.
+Function (and decorator) that converts a React component definition, React component class, or stand-alone render function, into a reactive component. A converted component will track which observables are used by its effective `render` and automatically re-render the component when one of these values changes.
+
+#### Functional Components
+
+`React.memo` is automatically applied to functional components provided to `observer`. `Observer` does not accept a functional component already wrapped in `React.memo`, or an `observer`, in order to avoid consequences that might arise as a result of wrapping it twice.
```

Comment:
Why would `Observer` have a problem with `React.memo` component? It does use a pure `useObserver` inside so it doesn't apply `React.memo`.

Reply:
Again, should be lower case.  I'm referring to your logger message:

> "Mobx observer: You are trying to use 'observer' on a function component wrapped in either another observer or 'React.memo'. The observer already applies 'React.memo' for you."

Reply:
Ah, that makes sense then :) Yea, it's a bit unfortunate to have two things which differ in first letter casing but have such a different behavior.